### PR TITLE
form added

### PR DESCRIPTION
**Pull request by adnan-khattak**
Ki There This is the form containing a splash or welcome screen and also sign in with a phone number and password sign up with the name, phone number, and password plz review it also needs some extra enhancement @Zakarya525

### CODE REVIEW

File: src/navigator/AppNavigator.js [hunk 1]
```diff
@@ -8,19 +8,21 @@ import InProgressComplaint from '../complaints/component/InProgressComplaint';
...
       <Stack.Screen name="signin" component={Signin} options={{headerShown:false}}/>
       <Stack.Screen name='signup' component={SignUp} options={{headerShown:false}}/>
       <Stack.Screen name='home' component={HomeScreen} options={{headerShown:false}} />
       <Stack.Screen name='Recieved' component={RecievedComplaint} options={{headerShown:false}} />
       <Stack.Screen name='complaint' component={ComplaintScreen} options={{headerShown:false}} />
       <Stack.Screen name='single' component={SingleComplaint} options={{headerShown:false}} />
       {/* <Stack.Screen name='tracking' component={ComplaintTracking} /> */}
+      <Stack.Screen name='welcome' component={Welcome} options={{headerShown:false}} />
     </Stack.Navigator>
   );
```
Comment (Zakarya525) [inline src/navigator/AppNavigator.js:27]: Great!! I suggest keeping all the private paths for authenticated users and auth screens for unauthenticated users. It is our initial step and it is better for now, once we integrate the API we should do this first... please R & D this thing for now.

File: src/screens/SignUp.js [hunk 2]
```diff
@@ -1,44 +1,69 @@
...
+        Already have an account?{' '}
+        <Text
+          style={user.clickHere}
+          onPress={() => navigation.navigate('signin')}>
+          Click here
+        </Text>
+      </Text>
     </View>
   );
 };
```
Comment (Zakarya525) [inline src/screens/SignUp.js:68]: are there any validations for forms (sign in and sign up) data? 🤐

Comment (Zakarya525) [reply src/screens/SignUp.js:68]: The answer is No. but no problem we can set the validations. please see 

1. react-hook-form
2. formik
3. js validation
4. react validations etc

File: src/screens/Welcome.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+      <View style={welcome.contentContainer}>
+      <Text style={welcome.appName}>Welcome To Community Cleanup App</Text>
+        {/* <Image source={require('../../assets/images/leaves.jpg')} style={welcome.logo} /> */}
+      </View>
+    </View>
+  );
+};
+
+
```
Comment (Zakarya525) [inline src/screens/Welcome.js:33]: I think Splash name will be more suitable for this screen, what do you think? 🙄

File: src/screens/Signin.js [hunk 4]
```diff
@@ -17,25 +19,36 @@ const Signin = ({navigation}) => {
...
+  </Text>
+</Text>
 
-      <TouchableOpacity style={user.button} onPress={handleSignin}>
-        <Text style={user.buttonText}>Sign In</Text>
-        {/* // adnan530@gmail.com addi3214 & adnan0@gmail.com kamranbhai addi12@gmail.com 12345612*/}
-      </TouchableOpacity> 
     </View>
   );
 };
```
Comment (Zakarya525) [inline src/screens/Signin.js:54]: please take care of validations.

**Review by Zakarya525 — COMMENTED** [review]
perfect. sure. great job overall. Just some improvements. InshaAllah

File: src/screens/SignUp.js (hunk 2 above)
Comment (Zakarya525) [reply src/screens/SignUp.js:68]: the answer is No. but no problem we have many things to do. please R&D these things:

- react-hook-form
- formik
- js validations etc
and we can also do this manually

**Comment by adnan-khattak** [comment]
Sure this is still in progress I have read your review and I m trying to do achieve all the stuff I.e validation etc.... ASAP in sha Allah

**Comment by Zakarya525** [comment]
please create PR only when your branch is ready for merge so that someone review it and then leave feedback for merge or improvement. Thanks 🙂

**Pull request closed without merging**